package.json

```
@@ -13,9 +13,9 @@
     "release": "node ./scripts/release",
     "build": "yarn --silent && tsc --build",
     "watch": "yarn --silent && tsc --build --watch",
-    "start": "cd packages/server && node dist/index.js",
-    "dev": "cd packages/server && yarn --silent && ts-node-dev --inspect --inspect-publish-uid=http --transpile-only src/index.ts",
-    "board": "cd packages/board && yarn dev",
+    "start": "yarn workspace @botpress/messaging-server start",
+    "dev": "yarn workspace @botpress/messaging-server dev",
+    "board": "yarn workspace @botpress/messaging-board dev",
```

We could maybe have these commands instead:

```suggestion
    "server:start": "yarn workspace @botpress/messaging-server start",
    "server:dev": "yarn workspace @botpress/messaging-server dev",
    "board:dev": "yarn workspace @botpress/messaging-board dev",
    ...
```

What do you think?

---

No I like the short commands. If we ever get to to many commands maybe, but 3 isn't hard to remember